core/src/main/java/io/paradoxical/cassieq/modules/LeadershipModule.java

```diff
@@ -0,0 +1,19 @@
+package io.paradoxical.cassieq.modules;
+
+import com.google.inject.AbstractModule;
+import io.paradoxical.cassandra.leadership.LeadershipStatusImpl;
+import io.paradoxical.cassandra.leadership.data.LeadershipSchema;
+import io.paradoxical.cassandra.leadership.factories.CassandraLeadershipElectionFactory;
+import io.paradoxical.cassandra.leadership.interfaces.LeadershipElectionFactory;
+import io.paradoxical.cassandra.leadership.interfaces.LeadershipStatus;
+
+public class LeadershipModule extends AbstractModule {
```

Comment:
we might want to consider having a `DefaultLeadershipModule` in the library. i'll add a issue for that.